jinahub/indexers/searcher/compound/NumpyFileSearcher/requirements.txt

```diff
@@ -1,2 +1,3 @@
 git+https://github.com/jina-ai/jina.git@010789331ce896ffcaea897402d698944f1a9d40
```

Comment:
use the tags

Reply:
That's Jina core. Shouldn't we install it with `jina==2.0.0rc9`?